packages/mdl-auto-init/README.md

```diff
@@ -0,0 +1,130 @@
+# MDL Auto Init
```

Comment:
It doesn't appear that it's possible to access component methods in auto-initialized components, with our current implementation of `mdl-auto-init`. If that's the case, this guid probably needs a section clearly explaining this limitation, or, on a broader note, explaining when it's recommended to use manual instantiation instead of `mdl-auto-init` (which is most of the time, really).